pom.xml

```diff
@@ -8,6 +8,42 @@
     <artifactId>datatools-server</artifactId>
     <version>3.2.0-SNAPSHOT</version>
 
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>https://opensource.org/licenses/MIT</url>
+        </license>
+    </licenses>
```

Comment:
Have you checked with all contributors with regards to this change of licensing?

Reply:
I don't think this is a change in license.  I'm just defining it here (the LICENSE file is [already defined](https://github.com/catalogueglobal/datatools-server/blob/dev/LICENSE)).

Reply:
Ah, perfect